jquery.flot.js

```diff
@@ -2710,6 +2645,92 @@ Licensed under the MIT license.
             }
         }
 
+        function seriesDataLimits(series) {
```

Comment:
multiple series can use the same axis ... I think you know that but just to be sure.

maybe computeRangeforDataSeries ?